modules/simple-payments/simple-payments.php

```diff
@@ -103,6 +102,7 @@ function parse_shortcode( $attrs, $content = false ) {
 		$data['id'] = $attrs['id'];
 		if ( ! wp_script_is( 'paypal-express-checkout', 'enqueued' ) ) {
 			wp_enqueue_script( 'paypal-express-checkout' );
+			wp_enqueue_style( 'simple-payments', plugins_url( '/simple-payments.css', __FILE__ ) );
```

Comment:
It's not necessary to add the `/` in front of the file name in `plugins_url`. It will be [ltrimmed anyway](https://github.com/WordPress/WordPress/blob/master/wp-includes/link-template.php#L3226).